securedrop/journalist_app/api.py

```diff
@@ -175,11 +175,7 @@ def remove_star(source_uuid: str) -> Tuple[flask.Response, int]:
     @api.route('/sources/<source_uuid>/flag', methods=['POST'])
     @token_required
     def flag(source_uuid: str) -> Tuple[flask.Response, int]:
-        source = get_or_404(Source, source_uuid,
-                            column=Source.uuid)
-        source.flagged = True
-        db.session.commit()
-        return jsonify({'message': 'Source flagged for reply'}), 200
+        return jsonify({'message': 'Sources no longer need to be flagged for reply'}), 200
```

Comment:
It may be worth marking the endpoint explicitly as deprecated, to be removed in a future release, and stating so in the docs as well (https://docs.securedrop.org/en/stable/development/journalist_api.html#flag-a-source).